src/executor.rs

```diff
@@ -274,3 +304,21 @@ impl CommandExt for Executor {
         self.spawn()
     }
 }
+
+fn log_real_command(c: &Command) {
+    println!(
+        "Executing {} {}",
+        c.get_program().to_string_lossy(),
+        c.get_args().map(|a| a.to_string_lossy()).collect::<Vec<_>>().join(" ")
+    );
+    let env = c.get_envs();
+    if env.len() != 0 {
+        println!(
+            "With env: {}",
+            env.filter(|(_, val)| val.is_some())
+                .map(|(key, val)| format!("{:?}={:?}", key, val.unwrap()))
+                .collect::<Vec<_>>()
+                .join(" ")
+        )
+    }
+}
```

Comment:
Why do we need this when we have `c.dry_run`? Can changes not be incorporated into `c.dry_run`?

Reply:
Since the directory is only added if it's `Some`, that's probably fine to add. If the env is also only added if it's explicitly added to the command construction (looks like it is, right?) then we can add both for completeness.

Reply:
Yeah, only added env is printed. Probably should have included an example of that in the initial comment.